config/kubernetes/org.yaml

```
@@ -395,6 +396,7 @@ members:
 - hongkailiu
 - honkiko
 - hoyho
+- howieyuen
```

missed this one in the previous review.

```suggestion
- howieyuen
- hoyho
```